modules/c/pages/gs-downloads.adoc

```diff
@@ -24,19 +24,29 @@ include::{root-partials}_show_get_started_topic_group.adoc[]
  
 .Downloads are available for the following versions:
 ****
+<<release-3-2-1>>   |
 <<release-3-2-0>>   |
 <<vs-release-1-0-0>>   |
 // | 
 ****
 
 // This block will always represent the major release version
+:param-version: 3.2.1
+:param-version-hyphenated: 3-2-1
+include::partial$downloadslist.adoc[]
+:param-version!:
+:param-version-hyphenated!:
+
+
 :param-version: 3.2.0
 :param-version-hyphenated: 3-2-0
-:vs-param-version: 1.0.0
-:vs-param-version-hyphenated: 1-0-0
```

Comment:
Oh... so 3.2.0 shouldn't show the downloads, only 3.2.1?

Reply:
Yep! Intended to prevent repeated inclusions of the 1.0.0 Vector Search downloads